generators/python/scripts/initialise.py

```diff
@@ -1,9 +1,12 @@
 import rospy
```

Comment:
we could remove import rospy here as it is added in the generation of test.py, right?